frontend/src/components/forms/EditLog.tsx

```diff
@@ -118,7 +118,7 @@ const EditLog = ({
       hour12: false,
     }),
   );
-  const [building, setBuilding] = useState("");
+  const [buildingId, setBuildingId] = useState<number>(logRecord.buildingId);
```

Comment:
so here you want to set the default to -1, and then in the `initializeValues` function you set the ID from the log record (which you've already done)